config/templates.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
     example: 'https://federalist-modern-team-template.18f.gov',
     title: 'Modern Team Template',
     description: 'Simple Jekyll site to showcase the work of an organization',
```

Comment:
I noticed that this one and the `USWDS Template` both mention `Jekyll` while the others do not. Do you think we should remove that word from these descriptions, too? Not necessary for merge, just a thought.

Reply:
I think we can certainly remove the work Jekyll from those descriptions or add the word Jekyll to the others. I'm in favor of removing the word Jekyll.

Reply:
I also am in favor of removing it

Reply:
Making it so ⚡️